jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpFields.java

```diff
@@ -989,6 +990,31 @@ default Mutable add(HttpFields fields)
             return this;
         }
 
+        /**
+         * Add a field with a list value;
+         *
+         * @param name the name of the field
+         * @param list the List value of the field. If null the field is cleared.
+         * @return this builder
+         */
+        default Mutable add(String name, List<String> list)
```

Comment:
If this is going to clash with `add(String, String)` when passing `null`, how about renaming it `addMulti()` to avoid the signature ambiguity?

Reply:
@sbordet  we'd need to rename the put(String name, List<String> list) as well...
So since we had the put version, I'm almost thinking this is OK... who would add explicit null?  Currently that would throw an IAE, so the code would not work anyway.